frontend/src/main/utils/systemInfo.js

```diff
@@ -12,6 +12,7 @@ export function useSystemInfo() {
       return {  
         springH2ConsoleEnabled: false,
         showSwaggerUILink: false,
+        sourceRepoUrl:"https://github.com/ucsb-cs156-s22/s22-4pm-courses",
```

Comment:
Minor nitpick: but can we change the default value here to the following?  That is a better long-term choice as a default:

```
https://github.com/ucsb-cs156/proj-courses
```